instructure/canvas-android
# [MBL-16911][Student] Assignments LGO

Test plan:
- Verify that no points and no total points are displayed when quantitative data is restricted.
- Verify that nothing changed when quantitative data is not restricted.

refs: MBL-16911
affects: Student
release note: none

## Checklist
